example/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "endpoint" {
```

Comment:
Is there a reason why we don't export the entire dict here?
For multi-node clusters we're losing information this way (nodes 2...n) and I think constructing an endpoint presumes that client libraries consume this in a certain way. Should we not leave this up to the user to consume as the see fit?

Reply:
dict won't work as a k8s secret, it expects a string; also the client library gets the address of the other nodes once connected via API